specs/airr-schema-openapi3.yaml

```diff
@@ -2148,11 +2202,15 @@ Diagnosis:
                         id: DOID:4
                         label: disease
         disease_length:
-            type: string
+            $ref: '#/PhysicalQuantity'
```

Comment:
See above -> `TimeInterval`

Reply:
Created TimeQuantity and made the change